src/const_eval.md

```diff
@@ -57,6 +57,22 @@ A _const context_ is one of the following:
   * [statics]
   * [enum discriminants]
 
+## Const Functions
+
+A _const fn_ is a function that one is permitted to call from a const context. Declaring a function
+`const` has no effect on any existing uses, it only restricts the types that arguments and the
+return type may use, as well as prevent various expressions from being used within it.
+
+Notable features that const contexts have, but const fn haven't are:
+
+* floating point types
```

Comment:
I'm a bit confused on this point. It seems that floating point types themselves are OK, it's floating-point operations that seem to be restricted?  That is, the following seems to be ok:

```rust
const fn foo(x: f32) -> f32 {
    x
}
```

Is it referring to something else?

Reply:
ah yes, float ops are forbidden, not values